Add pricing info everywhere (#45)
elastic/docs-content: serverless/partials/minimum-vcus-detail.mdx

<DocCallOut color="warning" title="Minimum runtime VCUs">

>     After you create an Elasticsearch Serverless project, to maintain basic Elasticsearch capabilities, a minimum number VCUs are always allocated to your project. These VCUs are used for the following purposes: 

Reviewer: ```suggestion
    When you create an Elasticsearch Serverless project, a minimum number of VCUs are always allocated to your project to maintain basic capabilities. These VCUs are used for the following purposes: 
```

>     After you create an Elasticsearch Serverless project, to maintain basic Elasticsearch capabilities, a minimum number VCUs are always allocated to your project. These VCUs are used for the following purposes: 

Reviewer 2: ```suggestion
    After you create an Elasticsearch Serverless project, to maintain basic Elasticsearch capabilities, a minimum number of VCUs are always allocated to your project. These VCUs are used for the following purposes: 
```

    - **Ingest**: Ensure constant availability for ingesting data into your project.
    - **Search**: Maintain a data cache and support low latency searches.

>     These minimum VCUs are billed at the standard rate per VCU hour.

Reviewer 2: ```suggestion
    These minimum VCUs are billed at the standard rate per VCU hour, incurring a minimum cost even when you're not actively using your project.
```
